src/platforms/android/server/platform.cpp

```diff
@@ -221,12 +223,14 @@ mg::PlatformPriority probe_graphics_platform(mo::ProgramOption const& /*options*
     hw_module_t const* hw_module;
 
     err = hw_get_module(HWC_HARDWARE_MODULE_ID, &hw_module);
-    if (err < 0) return mg::PlatformPriority::unsupported;
+    if (err < 0)
+        return mg::PlatformPriority::unsupported;
 
 #ifdef ANDROID_CAF
     // LAZY HACK to check for qcom hardware
-    if (strcmp(hw_module->author, "CodeAurora Forum") == 0)
-      return static_cast<mg::PlatformPriority>(mg::PlatformPriority::best + 1);
+    auto version = get_android_version();
```

Comment:
Would it be good to add a try block here? so if we cant for some reason run property_get?

Reply:
Like dlopen and dlsym property_get? So far we link against the library and I think the implementation does not throw.

Reply:
ok, will try tomorrow on and see if any of the devices break